scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml

```diff
@@ -112,7 +112,7 @@
           sleep 100 # time for the admin VM to boot
 
           # wipe out shared NFS that should be used by this deployment
-          ssh root@10.162.26.129 "/var/qa-scripts/wipe_nfs_shares.sh qa$hw_number"
+          ssh root@localhost "wipe_nfs_shares qa$hw_number"
```

Comment:
Yes, it is an option, but there is the overhead to configure such script for jenkins user: update sudoers file, add it to the jenkins user path. I'm still a bit concerned by giving jenkins user these sudo powers, it's there only for being a slave ;)